src/core/StelCore.hpp

```diff
@@ -305,6 +305,9 @@ class StelCore : public QObject
 
 	//! Return the observer heliocentric ecliptic position (GZ: presumably J2000)
 	Vec3d getObserverHeliocentricEclipticPos() const;
+	//! Return the observer heliocentric ecliptic velocity. This includes orbital and diurnal motion;
+	// diurnal motion is omitted if topocentric coordinates are in use.
```

Comment:
omitted if planetocentric coordinates are in use. (?)

Reply:
Oops, right.